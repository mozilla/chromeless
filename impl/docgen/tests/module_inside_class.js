/**
 * @class URL
 * A class which parses a url and exposes its various
 * components separately.
 */

/**
 * @module names_changed
 * This is a module who's wierd, it has its declaration
 * inside of a class declaration.  We could make this an error,
 * but why?
 */

/** @endclass */
