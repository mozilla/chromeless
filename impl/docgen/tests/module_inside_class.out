{
  "classes": [
    {
      "desc": "A class which parses a url and exposes its various\ncomponents separately.", 
      "name": "URL"
    }
  ], 
  "desc": "This is a module who's wierd, it has its declaration\ninside of a class declaration.  We could make this an error,\nbut why?", 
  "filename": "tests/module_inside_class.js", 
  "module": "names_changed"
}
